libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -22,12 +22,17 @@ class AP_BattMonitor_SMBus : public AP_BattMonitor_Backend
         BATTMONITOR_SMBUS_VOLTAGE = 0x09,              // Voltage
         BATTMONITOR_SMBUS_CURRENT = 0x0A,              // Current
         BATTMONITOR_SMBUS_REMAINING_CAPACITY = 0x0F,   // Remaining Capacity
-        BATTMONITOR_SMBUS_FULL_CHARGE_CAPACITY = 0x10, // Full Charge Capacity
+        BATTMONITOR_SMBUS_FULL_CHARGE_CAPACITY = 0x10, // Full Charge Capacity (accounting for battery degradation)
         BATTMONITOR_SMBUS_CYCLE_COUNT = 0x17,          // Cycle Count
+        BATTMONITOR_SMBUS_DESIGN_CAPACITY = 0x18,      // Design capacity (capacity when newly manufactured)
+        BATTMONITOR_SMBUS_DESIGN_VOLTAGE = 0x19,       // Design voltage
         BATTMONITOR_SMBUS_SPECIFICATION_INFO = 0x1A,   // Specification Info
+        BATTMONITOR_SMBUS_MANUFACTURE_DATE = 0x1B,     // Manufacture date
         BATTMONITOR_SMBUS_SERIAL = 0x1C,               // Serial Number
         BATTMONITOR_SMBUS_MANUFACTURE_NAME = 0x20,     // Manufacture Name
-        BATTMONITOR_SMBUS_MANUFACTURE_DATA = 0x23,     // Manufacture Data
+        BATTMONITOR_SMBUS_DEVICE_NAME = 0x21,          // Device Name
+        BATTMONITOR_SMBUS_DEVICE_CHEMISTRY = 0x22,     // Battery chemistry type
+        BATTMONITOR_SMBUS_MANUFACTURE_DATA = 0x23      // Manufacture Data
```

Comment:
Nitpick: leaving the comma doesn't break the compiler, and makes the diff smaller both for here, and future changes.